src/edr_pydantic/edr_feature_collection.py

```diff
@@ -0,0 +1,9 @@
+from typing import Dict
+
+from edr_pydantic.base_model import EdrBaseModel
+from edr_pydantic.parameter import Parameter
```

Comment:
Het moet `covjson_pydantic.parameter` zijn!

Totdat ze dat rechttrekken in de EDR spec. Ik dacht in v1.2?

Reply:
Fixed